system-tests/lib/cre/environment/environment.go

```diff
@@ -159,11 +143,11 @@ func SetupTestEnvironment(
 		}
 	}()
 
-	stageGen := NewStageGen(7, "STAGE")
+	stageGen := NewStageGen(6, "STAGE")
```

Comment:
Nit: consider putting the number in a global that NewStageGen increments every time it's called so you don't have to manage it manually